src/Dispatcher.cpp

```diff
@@ -264,10 +269,10 @@ void DispatcherBase::DispatchThreadMain() {
         // post outgoing messages if connection is active
         // only send keep-alives on client
         if (conn->state() == NetworkConnection::kActive)
-          conn->PostOutgoing(!m_server);
+          conn->PostOutgoing(m_networkMode & NT_NET_MODE_CLIENT);
```

Comment:
"only send keep-alives on client" implies to me an explicit check for client-ness

`!(m_networkMode & NT_NET_MODE_SERVER)` will also be true if the current mode is `NT_NET_MODE_NONE`